src/overlays/actors/ovl_Boss_Goma/z_boss_goma.c

```diff
@@ -1,15 +1,105 @@
 #include "z_boss_goma.h"
+#include "../ovl_En_Goma/z_en_goma.h"
 
 #define FLAGS 0x00000035
 
 #define THIS ((BossGoma*)thisx)
 
+/**
+ * IRIS_FOLLOW: gohma looks towards the player (iris rotation)
+ * BONUS_IFRAMES: gain invincibility frames when the player does something (throwing things?),
+ *                or randomly (see BossGoma_UpdateEye)
+ */
+typedef enum {
+    /**
+     * resets to this every Update
+     * allows not drawing lens and iris when eye is closed (see BossGoma_OverrideLimbDraw)
+     */
+    BOSSGOMA_EYESTATE_IRIS_FOLLOW_BONUS_IFRAMES,
+    BOSSGOMA_EYESTATE_IRIS_NO_FOLLOW_NO_IFRAMES,
+    BOSSGOMA_EYESTATE_IRIS_FOLLOW_NO_IFRAMES
```

Comment:
```suggestion
    EYESTATE_IRIS_FOLLOW_BONUS_IFRAMES,
    EYESTATE_IRIS_NO_FOLLOW_NO_IFRAMES,
    EYESTATE_IRIS_FOLLOW_NO_IFRAMES
```
I feel like because this is the only file this enum is used in these definitions can be shortened